activesupport/lib/active_support/number_helper/number_to_currency_converter.rb

```diff
@@ -12,8 +12,11 @@ def convert
         format = options[:format]
 
         if number.to_f.negative?
-          format = options[:negative_format]
           number = absolute_value(number)
```

Comment:
While we are here, looking to the implementation of this method it seems that `number` is always a string, correct? So why `absolute_value` is checking if it respond_to `abs`?

Reply:
Yes, the number is always a string. I'm not sure why `absolute_value` is checking for that. The method is private and only used once so I think it can be deleted and use the `sub` method directly to the number.